extensions/amp-bind/0.1/bind-impl.js

```diff
@@ -72,7 +72,7 @@ let BoundElementDef;
  */
 const BIND_ONLY_ATTRIBUTES = map({
   'AMP-CAROUSEL': ['slide'],
-  'AMP-LIST': ['state'],
+  'AMP-LIST': ['state', 'is-container'],
```

Comment:
keep a todo or issue somewhere to add this to validator protoascii file before removing experiment flag.